### DISCUSSION TITLE
Unable to Pull Items from Quickbooks Desktop

### DISCUSSION THREAD

**Discussion by hhelmric** [codatio/codat-docs#396] [Q&A] [answered]
First of all -- big shout out and thank you to the developers here! I am genuinely enjoying working with Codat -- and using the alpha TS SDK.

I believe I'm having a problem Pulling Items from Quickbooks Desktop. I was able to use the Codat API to create a new Item, and it did successfully Push to Quickbooks Desktop. That 1 Item shows up in the Codat portal as well as in response to requests to list Items via the Codat API. However, there were plenty of existing Items in Quickbooks Desktop that have not been Pulled into Codat, despite several Syncs.

Accounts, Invoices, Companies, etc have all Pulled in fine.

Is there something I'm missing about Items?

**Comment by jmyorston** [comment, marked as answer]
Hi @hhelmric 

Great to hear you are enjoying the TS SDK.

I can see the issue you are running into, at the moment you still have the default sync settings enabled, we do not have items enabled by default, but its easy to toggle this on.
You can follow the instructions [here](https://docs.codat.io/core-concepts/data-type-settings#override-the-default-sync-settings) to override the default settings and enable items to sync on a regular schedule.

**Reply by hhelmric** [reply]
Thanks! That worked
